src/ethereum/wallet.rs

```diff
@@ -70,6 +72,32 @@ impl Wallet {
         }
     }
 
+    pub fn private_key_from_seed(seed: &Seed) -> anyhow::Result<clarity::PrivateKey> {
+        let private_key = Self::root_extended_private_key_from_seed(seed)?
```

Comment:
:question: How do we know what `N` we are using when calling `root_extended_private_key_from_seed`? I am surprised that this compiles :open_mouth:

Reply:
:) The `N` does not matter to me so I am happy it compiles. I thought the same than you.